its/src/test/kotlin/org/sonarlint/intellij/its/fixtures/Editor.kt

```diff
@@ -31,12 +31,12 @@ import java.time.Duration
 @JvmOverloads
 fun ContainerFixture.editor(title: String, timeout: Duration = Duration.ofSeconds(20), function: Editor.() -> Unit = {}): ContainerFixture {
   find<ComponentFixture>(
-    byXpath("//div[@class='EditorTabs']//div[@accessiblename='$title' and @class='SingleHeightLabel']"), timeout).click()
+    byXpath("//div[@class='EditorTabs']//div[@accessiblename='$title' and (@class='SingleHeightLabel' or @class='EditorTabLabel')]"), timeout).click()
```

Comment:
Does that make it work on latest version? That's cool! Make sure to update https://sonarsource.atlassian.net/browse/SLI-1030, or in fact it might be easy to fix the ticket completely, up to you

Reply:
Yes, tested on CLion 2023.1.1, it works